ml-agents/mlagents/trainers/learn.py

```diff
@@ -450,8 +451,8 @@ def run_cli(options: RunOptions) -> None:
     except Exception:
         print("\n\n\tUnity Technologies\n")
     print(get_version_string())
-    trainer_logger = logging.getLogger("mlagents.trainers")
-    env_logger = logging.getLogger("mlagents_envs")
+    trainer_logger = create_logger("mlagents.trainers")
```

Comment:
How about letting `create_logger()` take the log level as an optional argument (and default to INFO)? That would simplify some of the code below.

Reply:
Actually, I think what you want to do is
* call `logging.basicConfig(level=logging.DEBUG)` or `logging.INFO` depending on `options.debug`
* don't set the logging level in `create_logger()`

I think if you do that, the new loggers will inherit the top-level log level.

Reply:
that makes sense. I'll add the log_level as an argument to the create_logger signature.

Reply:
the reason it has been hard to debug the logger initialization is that basicConfig was called twice in different files. given the order of import statements the config can be overridden. Hence I prefer to centralize the call to basicConfig in exactly one location and make sure its called exactly once per logger. I shall leave the basicConfig in logging_util.py